devicetypes/sky-nie/minoston-wallmote.src/minoston-wallmote.groovy

```diff
@@ -29,7 +29,7 @@
  */
 
 metadata {
-    definition (name: "Minoston Wallmote", namespace: "sky-nie", author: "winnie", ocfDeviceType: "x.com.st.d.remotecontroller", mcdSync: true) {
```

Comment:
I apologize, but this flag should actually should stay, my bad.

Reply:
Oh， I restore it.In fact, I don't understand the purpose of such concise flags.

Reply:
Code about associationGroup is not added. And in ST, button should support the value "pushed_3x", but  the vid of "generic-4-button" is not supported.

Reply:
> Code about associationGroup is not added.
You can add it, with check for a particular device, like here:
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/89a681dc155c037aa820d21fb31f06a1e7c8b350/devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy#L273

> And in ST, button should support the value "pushed_3x", but the vid of "generic-4-button" is not supported.
What do you mean by saying "vid of "generic-4-button" is not supported"? You've used it, by defining it in the fingerprint. You can do the same thing in another DTH. 
Or, if you've meant that "vid "generic-4-button" doesn't support `pushed_3x`", you can create your own presentation with our CLI, please see this guide:
https://community.smartthings.com/t/custom-capability-and-cli-developer-preview/197296
Especially `Generate / Post Device Configuration` part.

Reply:
@PKacprowiczS :
     Thanks! I have just completed the realization of these functions and have passed the test and acceptance.